README.md

```diff
@@ -16,7 +16,7 @@
 
 ## Background
 
-TBD
+For use with https://github.com/digitalbazaar/jsonld-signatures v6.0 and above.
```

Comment:
Is this still accurate or is 8.0.0 needed?

Reply:
@davidlehn - ah, thanks, I'll update to 8.